common/ast/src/ast/statement.rs

```diff
@@ -1035,6 +1073,70 @@ impl<'a> Display for Statement<'a> {
                 }
                 write!(f, " '{role}'")?;
             }
+            Statement::Grant(GrantStatement { source, principal }) => {
+                write!(f, "GRANT")?;
+                match source {
+                    GrantSource::Role { role } => write!(f, " ROLE {role}")?,
+                    GrantSource::Privs { privileges, level } => {
+                        write!(
+                            f,
+                            " {}",
+                            privileges
+                                .iter()
+                                .map(|p| p.to_string())
+                                .collect::<Vec<_>>()
+                                .join(", ")
+                        )?;
+                        write!(f, " ON")?;
+                        match level {
+                            GrantLevel::Global => write!(f, " *.*")?,
```

Comment:
Maybe this match parttern can extra to a function. But I'm not sure that is necessary.